libtextsecure/account_manager.js

```diff
@@ -532,12 +532,38 @@
         'private'
       );
 
+      /*
+      // Ensure that we always have a group conversation for comparisong
+      const debugConversation = await ConversationController.getOrCreateAndWait(
+        'debug',
+        'group'
+      );
+      await debugConversation.setFriendRequestStatus(4);
+      await debugConversation.set(updates);
+      */
+      const updates = {
+        name: 'GroupChat',
+        active_at: Date.now(),
+      };
+
+      // Ensure that we always have a public chat conversation for ourself
+      const publicChatConversation = await ConversationController.getOrCreateAndWait(
+        'lokiPublicChat',
+        'public'
+      );
+      updates.name = 'PublicChat';
```

Comment:
Why isn't this just set when updates is created above?

Reply:
Last minute shuffle when commenting out and committing